cdap-data-fabric/src/main/java/io/cdap/cdap/spi/data/nosql/NoSqlStructuredTableAdmin.java

```diff
@@ -82,6 +83,14 @@ public void createOrUpdate(StructuredTableSpecification spec)
     createTable(spec);
   }
 
+  @Override
+  public void createOrUpdate(List<StructuredTableSpecification> specs)
+      throws IOException, TableSchemaIncompatibleException {
+    for (StructuredTableSpecification spec : specs) {
+      createOrUpdate(spec);
+    }
+  }
```

Comment:
nit: Can this be a default implementation defined in the `StructuredTableAdmin.java` interface instead of having the same method in NoSql/Postgres etc... 

We already do something like this in multiread 
https://github.com/cdapio/cdap/blob/738a83ebad133acaaf8db24f7aac397fc31b3ad9/cdap-storage-spi/src/main/java/io/cdap/cdap/spi/data/StructuredTable.java#L99